tests/packages/packages/as/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "as",
+  "version": "0.0.2",
+  "description": "simple package with an ascMain",
+  "scripts": {
+    "test": "asp --reporter=SummaryTestReporter",
```

Comment:
Also `"test": "asp --reporter=SummaryTestReporter",` should be removed I guess

Reply:
ah yes good catch.  Weird that it still passed the tests.

Reply:
Ah it's because I replaced `npm run test` with `npm run build`.